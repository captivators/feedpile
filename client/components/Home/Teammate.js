import React from 'react';

const style = {
  gridList: {
    display: 'flex',
    width: '25%',
    overflowY: 'auto',
    marginBottom: '24px',
    textAlign: 'center',
  },
  porthole: {
    borderRadius: '100%',
    width: '75%',
    maxWidth: '200px',
    margin: '15px auto',
    display: 'block',
  }
}

const Teammate = (props) => {
  console.log('props', props)
  return (
    <div className="team-mate" style={style.gridList}>
      <img src={ props.info.image } style={style.porthole}/>
      { props.info.name }, <br />
      { props.info.job } <br />
    </div>
    )
}

export default Teammate;
