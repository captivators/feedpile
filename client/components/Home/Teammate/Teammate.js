import React from 'react';

const style = {
  gridList: {
    width: '25%',
    overflowY: 'hidden',
    marginBottom: '24px',
    fontFamily: 'Roboto, sans-serif',
    textAlign: 'center',
    fontFamily: 'Bitter',

  },
  porthole: {
    borderRadius: '100%',
    width: '75%',
    maxWidth: '150px',
    margin: '15px auto',
    imageAlign: 'center'
  }
}

const Teammate = (props) => {
  return (
    <div className="team-mate" style={style.gridList}>
      <img className="teammate-image" src={ props.info.image } style={style.porthole} alt=""/><br/>
      { props.info.name } <br />
      { props.info.job } <br />
    </div>
    )
}

export default Teammate;
