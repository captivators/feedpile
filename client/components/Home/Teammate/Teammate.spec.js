import React from 'react';
import { shallow, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Teammate from './Teammate'

test('Landingpage: Teammate take a snapshot', () => {
  const teammate= {
    info : {
      name: 'Faiz Mohammad',
      job: 'Software Engineer',
      image: 'https://s15.postimg.org/mnsiyo6ob/19718788.png'
    }
  }

  const component = shallow(
      <Teammate info={ teammate } />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});