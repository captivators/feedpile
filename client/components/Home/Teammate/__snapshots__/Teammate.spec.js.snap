// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landingpage: Teammate take a snapshot 1`] = `
<div
  className="team-mate"
  style={
    Object {
      "fontFamily": "Roboto",
      "marginBottom": "24px",
      "overflowY": "hidden",
      "textAlign": "center",
      "width": "25%",
    }
  }
>
  <img
    className="teammate-image"
    style={
      Object {
        "borderRadius": "100%",
        "imageAlign": "center",
        "margin": "15px auto",
        "maxWidth": "150px",
        "width": "75%",
      }
    }
  />
  <br />
   
  <br />
   
  <br />
</div>
`;
